pkg/internal/cnsoperator/cnsfilevolumeclient/v1alpha1/cnsfilevolumeclient_types.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2020 The Kubernetes authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// CnsFileVolumeClientSpec defines the desired state of CnsFileVolumeClient.
+type CnsFileVolumeClientSpec struct {
+	// ExternalIPtoClientVms maintains a mapping of External IP address to list of names of ClientVms.
+	// ClientsVms can be TKC Vms for TKC cluster (or) PodVms for SV.
+	// Keys are External IP Addresses that have access to a volume.
+	// Values are list of names of ClientVms. Each ClientVm in the list mounts this volume and
+	// is exposed to the external network by this IP address.
+	// PodVMs names will be prefixed with "PodVM" and TKC VMs will be prefix with "TKCVM".
```

Comment:
How about we use GUEST_VM to indicate any guest VM on SV cluster? This is a generic name that apples to TKC, SAP gardener, etc.